spec/menu_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'menu'
+
+describe Menu do
+  let(:dummy_cuisine_klass) {double :dummy_cuisine_klass}
+  let(:italian_menu) {double :italian_menu}
+  let(:dinner) {double :dinner}
+  let(:main) {double :main}
+  subject(:menu) {described_class.new dummy_cuisine_klass}
+  let(:default_menu_dinner_mains) {{ Mains: 'Price', Margherita: 13, Pasta: 12, Risotto: 11, Calzone: 10, Carbonara: 12, Bolognese: 12, Gnocchi: 11 }}
```

Comment:
Metrics/LineLength: Line is too long. [150/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)